gammapy/data/event_list.py

```diff
@@ -74,6 +76,30 @@ def read(cls, filename, **kwargs):
         table = Table.read(filename, **kwargs)
         return cls(table=table)
 
+    def write(self, filename, gti=None, overwrite=False):
+        """Write the event list to a FITS file.
+
+        If a GTI object is provided, it is saved into
+        a second extension in the file.
+        """
+
+        filename = make_path(filename)
+
+        if gti is None: # save just the events
+            if "extname" not in map(str.lower, self.table.meta.keys()):
+                self.table.meta = {"EXTNAME": "EVENTS"}
+            self.table.write(filename, overwrite=overwrite)
+
+        else:
+            assert isinstance(gti, GTI)
```

Comment:
Asserts should not be used in library code, better `raise TypeError('`gti` must be an instance of `GTI`')`